.github/workflows/upgrade-python-requirements.yml

```diff
@@ -2,79 +2,23 @@ name: Upgrade Requirements
 
 on:
   schedule:
-    # will start the job at 15:15 UTC every other week starting on the 5th of the month
-    - cron: "15 15 5/14 * *"
+     - cron: "15 15 5/14 * *"
   workflow_dispatch:
-    inputs:
-      branch:
-        description: "Target branch to create requirements PR against"
-        required: true
-        default: 'master'
-
+     inputs:
+       branch:
+         description: 'Target branch to create requirements PR against'
+         required: true
+         default: $default-branch
```

Comment:
@Jawayria this will not work. `$default-branch` is a template feature, not syntax which gets interpolated in general workflow files.
<img width="1439" alt="Screen Shot 2021-11-30 at 4 38 42 PM" src="https://user-images.githubusercontent.com/6405097/144132627-d1d9cc94-54f4-450b-881f-8fba4dac69dd.png"
<img width="1431" alt="Screen Shot 2021-11-30 at 4 40 31 PM" src="https://user-images.githubusercontent.com/6405097/144132720-25e2710f-6b30-4075-8855-4afc17b54870.png">
>
from https://github.com/edx/super-csv/runs/4373685050?check_suite_focus=true

Reply:
@matthugs Thanks so much for pointing it out. I tested this workflow in some repos before running cleanup job in bulk but this issue didn't come up as we set branch name to 'master' [here](https://github.com/edx/.github/blob/18d8eb85e21f424c1600922f0d9279770cc683f7/.github/workflows/upgrade-python-requirements.yml#L51) if input is empty and most of the repos had 'master' as default branch.
I've started propagating this correction in other repos as well.